src/brevitas_examples/imagenet_classification/ptq/ptq_evaluate.py

```diff
@@ -158,16 +158,44 @@ def validate_args(args):
     default=20,
     type=int,
     help='Numbers of iterations for graph equalization (default: 20)')
+parser.add_argument(
+    '--learned-round',
+    default=None,
+    type=str,
+    choices=[None, 'linear_round', 'hard_sigmoid_round', 'sigmoid_round'],
+    help='Learned round type (default: None)')
+parser.add_argument(
+    '--learned-round-loss',
+    default='regularised_mse',
+    type=str,
+    choices=['regularised_mse', 'mse'],
+    help='Learned round type (default: none)')
+parser.add_argument(
+    '--learned-round-mode',
+    default='layerwise',
+    choices=['layerwise', 'blockwise'],
+    help='Learned round mode (default: none)')
 parser.add_argument(
     '--learned-round-iters',
     default=1000,
     type=int,
     help='Numbers of iterations for learned round for each layer (default: 1000)')
+parser.add_argument(
+    '--learned-round-lr-scheduler',
+    default=None,
+    type=str,
+    choices=[None, 'linear'],
+    help='Learning rate scheduler for learned round (default: None)')
 parser.add_argument(
     '--learned-round-lr',
     default=1e-3,
     type=float,
     help='Learning rate for learned round (default: 1e-3)')
+parser.add_argument(
+    '--learned-round-batch-size',
+    default=1,
```

Comment:
Is this consistent with past behaviour?

Reply:
Yes, previously the learned round batch size was fixed to 1